cmd/main.go

```diff
@@ -79,14 +92,9 @@ func init() {
 	cobra.OnInitialize()
 
 	rootCmd.Flags().AddGoFlagSet(flag.CommandLine)
-	rootCmd.Flags().StringVar(&endPoint, "endpoint", "unix://tmp/csi.sock", "CSI endpoint")
-	rootCmd.Flags().StringVar(&driverName, "drivername", string(operatorv1.SharedResourcesCSIDriver), "name of the driver")
-	rootCmd.Flags().StringVar(&nodeID, "nodeid", "", "node id")
-	rootCmd.Flags().Int64Var(&maxVolumesPerNode, "maxvolumespernode", 0, "limit of volumes per node")
-	rootCmd.Flags().StringVar(&shareRelistInterval, "share-relist-interval", "",
-		"the time between controller relist on the share resource expressed with golang time.Duration syntax(default=10m")
+	rootCmd.Flags().StringVar(&cfgFilePath, "config", "/var/run/configmaps/config/config.yaml", "configuration file path")
+	rootCmd.Flags().StringVar(&nodeID, "nodeid", "", "Kubernetes node identification")
```

Comment:
My [previous comment](https://github.com/openshift/csi-driver-shared-resource/pull/60#discussion_r730694753) addresses the configuration attributes, what stays as flags and what goes to the configuration file, please consider.

> We either
> 
>     1. always just read from the config map
> 
>     2. always depend on the csi-driver-shared-resource-operator to map into changes to our DaemonSet and the command line options specfied there ... i.e. https://github.com/openshift/csi-driver-shared-resource/blob/c8dab4fb90277123868abe8f05086fa48ed1f2d5/deploy/csi-hostpath-plugin.yaml#L68-L96
> 
> 
> My initial thought is 1) is better, but not having done the end to end coding on this you have, I'll defer to you if you have insight based on your prototyping that says otherwise.

I agree option `1` is better, it's also the subject of latest changes as well. The configuration file is always mounted from the `ConfigMap` rolled out with the driver, when it changes, the driver is restarted too.